src/App.jsx

```diff
@@ -0,0 +1,31 @@
+import './App.css'
+
+function App() {
+
+  const genLinks = () =>{
+    let arr = []
+    for(let i = 0; i< 10; i++){
+      arr.push({text : `Exercise ${i}`, link: `/exercise${i}`})
+    }
+    return arr;
+  }
+
+  const array = genLinks()
+  const array1 = ['dog', 'cat', 'chicken', 'cow', 'sheep', 'horse']
```

Comment:
It would be better to name these arrays respective to the information they hold for better code readability. For example,
```suggestion
  const links = genLinks()
  const animals = ['dog', 'cat', 'chicken', 'cow', 'sheep', 'horse']
```